usaspending_api/awards/serializers_v2/serializers.py

```diff
@@ -1,4 +1,8 @@
 from rest_framework import serializers
+from usaspending_api.awards.models import Award, TransactionFPDS
+from usaspending_api.references.models import (Agency, LegalEntity, Location, LegalEntityOfficers,
+                                               SubtierAgency, ToptierAgency, OfficeAgency)
+# from usaspending_api.recipient.models import DUNS
```

Comment:
Can be removed since the rest of the code section which needed this should be removed